src/pages/SignupPage.tsx

```diff
@@ -1,3 +1,217 @@
+import Body from '@ui/components/layout/Body';
+import styled from '@emotion/styled';
+import styleTokenCss from '@ui/styles/styleToken.css';
+import { ChangeEvent, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router';
+import { PATH } from '@lib/const/path';
+import { User, UserValidation } from '@lib/types/user';
+import NavigationHeader from '@ui/components/layout/NavigationHeader';
+import getValidationUser from '@lib/utils/getValidationUser';
+import SignButton from '@ui/components/SignButton';
+import InputBox from '@ui/components/InputBox';
+
 export default function SignupPage() {
-  return <>signup page</>;
+  const [user, setUser] = useState<User>({
+    email: '',
+    password: '',
+    passwordCheck: '',
+    name: '',
+  });
+  const [userValidation, setUserValidation] = useState<UserValidation>({
+    email: false,
+    password: false,
+    passwordCheck: false,
+    name: false,
+  });
+  const [isChecked, setIsChecked] = useState(false);
+  const navigate = useNavigate();
+
+  const handleChangeUser = (e: ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    let test = getValidationUser(name as 'email' | 'password' | 'name', value);
+
+    if (name === 'passwordCheck' && user.password === value) {
+      test = true;
+    }
```

Comment:
```suggestion
  const { name, value } = e.target;
  let test = false;
  if (name === 'passwordCheck') {
    test = user.password === value;
  } else {
    test = getValidationUser(name as 'email' | 'password' | 'name', value);
  }
```

P2:
passwordCheck는 getValidationUser 함수를 거칠 필요가 없어보여요.
위 처럼 if문으로 분기해서 로직을 작성해주면 가독성 면에서도 어떤 값을 대입해주는지 명확해질것 같습니다.

Reply:
이렇게 작성하면 코드가 좀 더 보기에 명확해지네요. 👀 반영하겠습니다.